modules/currency.js

```diff
@@ -335,3 +346,27 @@ export function setOrtbCurrency(ortbRequest, bidderRequest, context) {
 }
 
 registerOrtbProcessor({type: REQUEST, name: 'currency', fn: setOrtbCurrency});
+
+export const requestBidsHook = timedAuctionHook('currency', function requestBidsHook(fn, reqBidsConfigObj) {
+  const hookConfig = {
+    reqBidsConfigObj,
+    context: this,
+    nextFn: fn,
+    haveExited: false,
```

Comment:
this is not used anywhere. Probably a typo and it was meant to be `hasExited` (which would also be falsy if left undefined).